config/longrun_configs/amip_diagedmf_10years.yml

```diff
@@ -0,0 +1,25 @@
+FLOAT_TYPE: "Float32"
+albedo_model: "CouplerAlbedo"
+atmos_config_file: "config/longrun_configs/amip_target_diagedmf.yml"
+bucket_albedo_type: "map_temporal"
+checkpoint_dt: "366days"
+co2_model: "maunaloa"
+coupler_toml: ["toml/amip.toml"]
```

Comment:
@akshaysridhar do we want to use `"toml/amip_target_topo_diagedmf.toml"` instead?